src/App.js

```diff
@@ -9,20 +9,30 @@ export default function App() {
   const [contacts, setContacts] = useState([])
   
   //TODO: Load all contacts on useEffect when component first renders
+  useEffect(function(){
+    fetch('http://localhost:4000/contacts')
+    .then(response => response.json())
+    .then(json => {
+      setContacts(json)
+    })
+  }, [])
 
   return (
     <>
       <nav>
         <h2>Menu</h2>
         <ul>
           {/* TODO: Make these links */}
-          <li>Contacts List</li>
-          <li>Add New Contact</li>
+          <li><Link to="/">Contacts List</Link></li>
+          <li><Link to="/contacts/add">Add New Contact</Link></li>
         </ul>
       </nav>
       <main>
         <Routes>
           {/* TODO: Add routes here  */}
```

Comment:
Room for improvement: would've been great to see your own comments, explaining on what the code is doing.